automatminer/examples/mse_example.py

```diff
@@ -0,0 +1,45 @@
+import unittest
```

Comment:
Looks good. Also is cool that it's in a test, but I'm afraid it will just confuse people who come to use it (ie, "woah, it is weird they put this test here, I wonder where the example is"). I don't think we will be frequently running this test anyway (_especially_ because you are using the default config, not the debug config which is much faster).

Reply:
Instead of being in a test, could you write out the file in a simple script (or even better, **notebook**), with perhaps a few more comments? Not everyone is particularly familiar with pandas/automatminer/matminer/machine learning, and there are a few areas that won't make sense to someone unfamiliar with this stack.

For example, when we are renaming the formula column, just add a comment saying "The preset automatminer uses pre-defined column names 'composition' and 'structure' to find the composition and structure columns. You can change these by editing your config"